docs/user-guide/gpudeduplication.rst

```diff
@@ -184,7 +184,7 @@ Python API
     from nemo_curator import FuzzyDuplicatesConfig
 
     config = FuzzyDuplicatesConfig(
-        cache_dir="/path/to/dedup_outputs",
+        cache_dir="/path/to/dedup_outputs", # must be cleared between runs
```

Comment:
Line 254 has the tip already, can we add somewhere else too?

Reply:
I believe other PR's are tackling some of the documentation changes. The only other place I can think of is: https://github.com/NVIDIA/NeMo-Curator/blob/main/examples/fuzzy_deduplication.py